<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PLRENT-8BFM4H2" failures="1" tests="9" name="com.axelerant.testcases.TestCases_AccountOpening" time="33.471" errors="0" timestamp="2021-11-30T18:27:41 IST" skipped="0">
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_login" time="2.583"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_login
Logging into john's account
Entering text john in  field
Entering text demo in  field
John Smith logged in successfully
Test Execution PASSED for -> tc_login]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_CheckingAccountOpening" time="3.962"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_CheckingAccountOpening
Opening new Checking account
Checking account open successfully. Account ID is 28107
Test Execution PASSED for -> tc_CheckingAccountOpening]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_VerifyCheckingsAccountDetails" time="3.382"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_VerifyCheckingsAccountDetails
Verifying Checking account details page
Checking Account number is : 28107
Account type is : CHECKING
Account Balance is : $100.00
Test Execution PASSED for -> tc_VerifyCheckingsAccountDetails]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_SavingsAccountOpening" time="3.586"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_SavingsAccountOpening
Opening new Savings account
Savings account open successfully. Account ID is 28218
Test Execution PASSED for -> tc_SavingsAccountOpening]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_VerifySavingsAccountDetails" time="3.107"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_VerifySavingsAccountDetails
Verifying Savings account details page
Savings Account number is : 28218
Account type is : SAVINGS
Account Balance is : $100.00
Test Execution PASSED for -> tc_VerifySavingsAccountDetails]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_validateErrorMessages" time="8.681"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_validateErrorMessages
Successfully navigated to Bill Payment services screen.
Validate fields on Bill pay form
Fetching all error strings from the website
Fetching all error strings from the website
[Payee name is required., Address is required., City is required., State is required., Zip Code is required., Phone number is required., Account number is required., Account number is required., The amount cannot be empty.]
Error messages matching with expected copy
Check error message for invalid account number
Entering text BRlLg in  field
Please enter a valid number.
Correct error message dissplayed for invalid account number
Entering text ykZMe in  field
Please enter a valid number.
Correct error message displayed for mismatch in account number
Check Error message for incorrect account number in Account# and Verify Account# field
Entering text 58411 in  field
Entering text 55100 in  field
The account numbers do not match.
Correct error message displayed for mismatch in account number
Validate Ammount field
Entering text BEznj in  field
Please enter a valid amount.
Correct error message dissplayed for invalid account number
Test Execution PASSED for -> tc_validateErrorMessages]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_transferFunds" time="2.596"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_transferFunds
Transfering funds from savings to checking account
Entering text DSHxRVNAaM in  field
Entering text RDECOycHBSkyvKidLIVJ in  field
Entering text fCfgY in  field
Entering text sFOzZEHDbg in  field
Entering text aicmaV in  field
Entering text 1039446839 in  field
Entering text 28107 in  field
Entering text 28107 in  field
Entering text 200 in  field
Bill payment successfull
Test Execution PASSED for -> tc_transferFunds]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_checkDeductionFromSavings" time="2.694"/>
  <system-out>
    <![CDATA[Test Execution started for -> tc_checkDeductionFromSavings
Checking the right amout is deducted from Savings balance
Navigating to URL - https://parabank.parasoft.com/parabank/activity.htm?id=28218
Last transaction is : $200.00
Transaction amount deducted is correct
Previous balance was $100.00
Previous balance is -$100.00
Savings account balance is deducted by $200.00
Test Execution PASSED for -> tc_checkDeductionFromSavings]]>
  </system-out>
  <testcase classname="com.axelerant.testcases.TestCases_AccountOpening" name="tc_checkAdditionToChecking" time="2.880">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.axelerant.testcases.TestCases_AccountOpening.tc_checkAdditionToChecking(TestCases_AccountOpening.java:263)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- tc_checkAdditionToChecking -->
  <system-out>
    <![CDATA[Test Execution started for -> tc_checkAdditionToChecking
Checking the right amout is Added to Checking balance
Navigating to URL - https://parabank.parasoft.com/parabank/activity.htm?id=28107
Last transaction is : 
Transaction amount is not added
Test Execution FAILED for -> tc_checkAdditionToChecking]]>
  </system-out>
</testsuite> <!-- com.axelerant.testcases.TestCases_AccountOpening -->
